adam-cli/pom.xml

```diff
@@ -14,22 +14,35 @@
   <build>
     <plugins>
       <plugin>
-        <groupId>org.codehaus.mojo</groupId>
-        <artifactId>appassembler-maven-plugin</artifactId>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-shade-plugin</artifactId>
         <configuration>
-          <programs>
-            <program>
-              <mainClass>org.bdgenomics.adam.cli.ADAMMain</mainClass>
-              <id>adam</id>
-            </program>
-          </programs>
+          <createDependencyReducedPom>true</createDependencyReducedPom>
+          <filters>
+            <filter>
+              <artifact>*:*</artifact>
+              <excludes>
+                <exclude>META-INF/*.SF</exclude>
+                <exclude>META-INF/*.DSA</exclude>
+                <exclude>META-INF/*.RSA</exclude>
+              </excludes>
+            </filter>
+          </filters>
```

Comment:
@heuermh Did you see any changes that needed to by made to the PR when you looked at the old `maven-shade-plugin` xml?

This RP looks good to me but, since it's a big change (and I've hit the merge button too quickly a few times lately), I wanted to check.